git clone https://github.com/pangeo-data/pangeo-docker-images
cd pangeo-docker-images
git checkout 2021.06.05
patch -p1 < $this_file
cd base-image
docker build -t mwa4/base-image:1 .
cd ../base-notebook
docker build -t mwa4/base-notebook-pangeo:1 .

diff --git a/base-image/Dockerfile b/base-image/Dockerfile
index ef99294..58d26c0 100644
--- a/base-image/Dockerfile
+++ b/base-image/Dockerfile
@@ -1,4 +1,5 @@
-FROM ubuntu:20.04
+# docker build -t mwa4/base-image:1 .
+FROM debian:testing
 # build file for pangeo images
 
 LABEL org.opencontainers.image.source=https://github.com/pangeo-data/pangeo-docker-images
diff --git a/base-notebook/Dockerfile b/base-notebook/Dockerfile
index e34f8fb..9d6c805 100644
--- a/base-notebook/Dockerfile
+++ b/base-notebook/Dockerfile
@@ -1 +1,2 @@
-FROM pangeo/base-image:master
+# docker build -t mwa4/base-notebook-pangeo:1 .
+FROM mwa4/base-image:1
